pkgs/applications/networking/cluster/fluxcd/default.nix

```diff
@@ -65,8 +71,10 @@ in buildGoModule rec {
       updates to configuration when there is new code to deploy.
     '';
     homepage = "https://fluxcd.io";
+    downloadPage = "https://github.com/fluxcd/flux2/releases/tag/v${version}";
     license = licenses.asl20;
     maintainers = with maintainers; [ bryanasdev000 jlesquembre ];
+    platforms = platforms.all;
```

Comment:
```suggestion
```

It is automatically set by  `buildGoModule` (especially because not `all` platforms can run Go).

Reply:
Done!